src/main/windows/settings.ts

```diff
@@ -49,7 +49,22 @@ ipcMain.on(
     if (settingsWindow) {
       // Not truncating the height makes this function throw
       // in windows
-      settingsWindow.setBounds({ height: Math.trunc(height) });
+      settingsWindow.setBounds(
+        {
+          height: Math.trunc(height),
+        },
+        true
+      );
     }
   }
 );
+
+ipcMain.handle('dark-mode:light', () => {
+  nativeTheme.themeSource = 'light';
```

Comment:
I believe we also need to set the theme when the app starts, not found any clues in the documentation but after setting the dark theme and restarting the app (on dev) the theme was back to light even when the config correctly loaded the dark one.
![image](https://github.com/internxt/drive-desktop/assets/38570230/1e22657c-527d-4547-867d-9d62745b9ac0)

Reply:
Good catch! I'll look into this and make the theme update on start 👍🏻